03 - Data Science and Custom Skills/Azure Function Skills/SampleSkills/1_BingEntitySearchSkill.cs

```diff
@@ -0,0 +1,400 @@
+                    "description": "Pablo Ruiz Picasso was a Spanish painter, sculptor, printmaker, ceramicist, stage designer, poet and playwright who spent most of his adult life in France. Regarded as one of the most influential artists of the 20th century, he is known for co-founding the Cubist movement, the invention of constructed sculpture, the co-invention of collage, and for the wide variety of styles that he helped develop and explore. Among his most famous works are the proto-Cubist Les Demoiselles d'Avignon, and Guernica, a dramatic portrayal of the bombing of Guernica by the German and Italian airforces during the Spanish Civil War.",
+                    "imageUrl": "https://www.bing.com/th?id=AMMS_e8c719d1c081e929c60a2f112d659d96&w=110&h=110&c=12&rs=1&qlt=80&cdv=1&pid=16.2",
+                    "url": "http://en.wikipedia.org/wiki/Pablo_Picasso",
+                    "licenseAttribution": "Text under CC-BY-SA license",
+                    "entities": { ... }
+                 }
+            },
+            ...
+        ]
+    }
+  
+ *****************************************************/
+
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+
+
+namespace SampleSkills
+{
+    /// <summary>
+    /// Sample custom skill that wraps the bing entity search API to connect it with a 
+    /// congitive search pipeline.
+    /// </summary>
+    public static class BingEntitySearch
+    {
+        #region Credentials
+        // IMPORTATNT: Make sure to enter your credential and to verify the API endpoint matches yours.
+        static string bingApiEndpoint = "https://api.cognitive.microsoft.com/bing/v7.0/entities/";
+        static string key = "";  
+        #endregion
+
+        #region Class used to deserialize the request
+        public class InputRecord
+        {
+            public class InputRecordData
+            {
+                public string name;
+            }
+
+            public string recordId { get; set; }
+            public InputRecordData data { get; set; }
+        }
+
+        private class WebApiRequest
+        {
+            public List<InputRecord> values { get; set; }
+        }
+        #endregion
+
+        #region Classes used to serialize the response
+        public class OutputRecord
+        {
+            public class OutputRecordData
+            {
+                public string name;
+                public string description;
+                public string imageUrl;
+                public string url;
+                public string licenseAttribution;
+                public Entities entities { get; set; }
+            }
+
+            public class OutputRecordErrors
```

Comment:
Same comment here regarding using the same class for errors/warnings https://github.com/Microsoft/AzureSearch_JFK_Files/blob/master/JfkWebApiSkills/JfkWebApiSkills/WebApiSkill/WebAPISkillContract.cs